html/screensharingtest.js

```diff
@@ -22,13 +24,13 @@ var spinner = null;
 
 // Just an helper to generate random usernames
 function randomString(len, charSet) {
```

Comment:
I think we can get rid of this function entirely. since we have `Janus.randomString` already (which incidentally is used just a few lines above). Possibly unrelated to the other changes, but since we're cleaning up...

Reply:
Ok, I am removing it in the next commit.

Reply:
oh it looks like it is being used in the script
https://github.com/meetecho/janus-gateway/blob/961b5764b34b5f84c56825f8acf05890d4551aa4/html/screensharingtest.js#L380
https://github.com/meetecho/janus-gateway/blob/961b5764b34b5f84c56825f8acf05890d4551aa4/html/screensharingtest.js#L419

Reply:
Yes but we can just use Janus.randomString() where the other function is used, they're exactly the same function.